Demo/RCHAccountViewController.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  RCHAccountViewController.swift
+//  RichRelevanceSDKDemo
+//
+//  Created by Ariana Antonio on 10/6/16.
+//  Copyright © 2016 Raizlabs Inc. All rights reserved.
+//
+
+import UIKit
+
+class RCHAccountViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet var accountTableView: UITableView!
+    @IBOutlet var userIDTextField: UITextField!
+    @IBOutlet var modalView: UIView!
+    @IBOutlet var saveUserButton: UIButton!
+    @IBOutlet var userCellLabel: UILabel!
+    
+    var userIDArray: [String]?
+    var currentUser: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationController?.navigationBar.tintColor = UIColor.white
+        saveUserButton.layer.cornerRadius = 4
+        modalView.isHidden = true
+        
+        guard let userIDArray = UserDefaults.standard.object(forKey: kRCHUserDefaultKeyUserIDs) as? [String] else {
+            self.userIDArray = []
+            return
+        }
+        self.userIDArray = userIDArray
+        currentUser = UserDefaults.standard.string(forKey: kRCHUserDefaultKeyCurrentUser)
+    }
+    
+    // MARK: Actions
+    
+    @IBAction func addUserSelected(_ sender: AnyObject) {
+        modalView.isHidden = false
+    }
+    
+    @IBAction func saveUserSelected(_ sender: AnyObject) {
+        modalView.isHidden = true
+        view.endEditing(true)
+        guard let newUser = userIDTextField.text else {
+            return
+        }
+        saveNewUser(withUser: newUser)
+    }
+    
+    @IBAction func dismissModalSelected(_ sender: AnyObject) {
+        modalView.isHidden = true
+        view.endEditing(true)
+    }
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        view.endEditing(true)
+        return true
+    }
+    
+    func saveNewUser(withUser newUser: String) {
+        let mutableUserArray: NSMutableArray = []
+        if let userArray = UserDefaults.standard.object(forKey: kRCHUserDefaultKeyUserIDs) as? [String] {
+            mutableUserArray.addObjects(from: userArray)
+        }
+        mutableUserArray.add(newUser)
```

Comment:
You should always try to avoid `NSMutable`'s if possible. Does this work?
```
var users = UserDefaults.standard.object(forKey: kRCHUserDefaultKeyUserIDs) as? [String] ?? []
users.append(newUser)
```